shared/transforms/shared_shorthand2xccdf.xslt

```diff
@@ -394,7 +394,6 @@
               <!-- add clauses if specific macros are found within -->
               <xsl:if test="sysctl-check-macro">the correct value is not returned</xsl:if>
               <xsl:if test="fileperms-check-macro or fileowner-check-macro or filegroupowner-check-macro">it does not</xsl:if>
-              <xsl:if test="partition-check-macro">no line is returned</xsl:if>
```

Comment:
These strings will be missing in OCIL. It creates  a weird question in OCIL: `Is it the case that ?`

Reply:
Indeed, this is an issue that has to be solved somehow before merging this PR.

Reply:
Resolved. I have rebased to keep the mapping commit <-> template converted.